src/SIL.XForge.Scripture/Services/SFProjectService.cs

```diff
@@ -614,8 +617,6 @@ public async Task<string> CheckLinkSharingAsync(string curUserId, string shareKe
 
         string projectId = projectSecret.Id;
         ShareKey projectSecretShareKey = projectSecret.ShareKeys.FirstOrDefault(sk => sk.Key == shareKey);
-        if (projectSecretShareKey == null)
```

Comment:
Was this removed because it will throw an exception anyway if the share key is null?